python/services/openai.py

```diff
@@ -27,7 +26,7 @@ def generate_cover_letter(raw_job_details, best_match_section,api_key):
 
 Additional Instructions:
     1. Address the letter to the hiring manager
-    2. Use the candidate's name as "Umair Jibran"
+    2. Use the candidate's name from CV
```

Comment:
aah, good catch 😅

Sorry I'm sure it would've made you cry to see where the name is coming from :D